converter.go

```diff
@@ -7,6 +7,7 @@ import (
 	"database/sql/driver"
 	"encoding/hex"
 	"fmt"
+	"github.com/apache/arrow/go/arrow/memory"
```

Comment:
nit: can this be separated out by a newline apart from the other dependencies?

Reply:
whops that slipped in there! thanks :)